uxarray/remap/inverse_distance_weighted.py

```diff
@@ -231,12 +231,13 @@ def _inverse_distance_weighted_remap_uxda(
     )
     # add remapped variable to existing UxDataset
     if isinstance(destination_obj, uxarray.core.dataset.UxDataset):
-        destination_obj[source_uxda.name] = uxda_remap
-        return destination_obj
+        uxds = destination_obj.copy()
```

Comment:
See my comment below, but we do not want to do this.

Reply:
> I do remember you mentioning only allowing a grid, I figured I would fix that in a different PR and get this patched before the notebook is pushed. But I can tackle it here,

Would be nice to tackle it here. Let me know if you need any help next week.

> Is there ever a case where I want to take and add a second variable to a data array using a remap?

The nice thing about a Data Array is that it only stores a single data variable. A Dataset is used for one or more data variables. 

This does bring up the point of being able to do a "Dataset-wide" remap, i.e. apply the remapping on every variable in a `UxDataset`

> How would I be able to take and say add two remapped variables from two different datasets together in one data array?

We'd use the typically Xarray interface to merge one or more data arrays (or datasets) into a new dataset. 

https://docs.xarray.dev/en/stable/user-guide/combining.html#merge